pylint/lint/expand_modules.py

```diff
@@ -48,7 +48,6 @@ def expand_modules(
     result: List[ModuleDescriptionDict] = []
     errors: List[ErrorDescriptionDict] = []
     path = sys.path.copy()
-
```

Comment:
Personally I like these white lines for readability, but I think that is personal preference.

Reply:
Added back :)